packages/icons/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @giosg-design-system/icons
 
+## 2.0.0
+
+### Major Changes
+
+- 9d47b7d: new icon
```

Comment:
i think it should be a minor change instead. because we are not introducing any breaking changes